test/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1152,7 +1152,7 @@ public void TestGroupBy()
                 if (originalColumn.Name == "Bool")
                     continue;
                 DataFrameColumn headColumn = head.Columns[originalColumn.Name];
-                Assert.Equal(originalColumn[5], headColumn[verify[5]]);
+                Assert.Equal(originalColumn[7], headColumn[verify[5]]);
```

Comment:
`5` -> `7` because `null` is equivalent to `default` in the `GroupBy` method anymore